### DISCUSSION TITLE
[edgedb-cli] Migrations are applied automatically on Cloud when you acces the instance throw the CLI.

### DISCUSSION THREAD

**Discussion by Mat-moran** [geldata/gel#5800] [Q&A] [answered]
Hi,

I am using the early acces cloud instance and I notice that migrations are applied automatically when you acces the instance throw the CLI. The command I have been using to connect to the cloud is 
`edgedb project init --server-instance user/instance`

Is this behavieour expected? 
If this is expected, IMO would be great if you can decide to apply the migrations or not. 

Thanks 😄

**Comment by elprans** [comment, marked as answer]
This is intended behavior of `project init`.  If you want to skip migrations, pass `--no-migrations`
